### DISCUSSION TITLE
视频倍速记忆和扩展倍速菜单可以分开做成独立组件吗

### DISCUSSION THREAD

**Discussion by SeanX16** [the1812/Bilibili-Evolved#2723] [功能建议 / Ideas]
目前版本里，扩展倍速菜单这个功能是被放到了倍速记忆组件里面的，所以我想扩展倍速菜单就必须开启倍速记忆，没法单独开启。希望能修改一下，谢谢啦！
![image](https://user-images.githubusercontent.com/69029679/146160721-27e9ebc3-0d37-41ae-840c-215ad8d7964b.png)

**Comment by Geeyun-JY3** [comment]
可以学会在开讨论前先善用搜索吗：#2623、#2660

**Reply by SeanX16** [reply]
没怎么用过Github，不好意思了。。。我删贴吧

**Reply by SeanX16** [reply]
这能删吗

**Reply by Geeyun-JY3** [reply]
不用删，下次注意就行

**Reply by SeanX16** [reply]
嗯嗯
